pipeline/pipeline.go

```diff
@@ -1,22 +1,41 @@
 package pipeline
 
-import "sort"
+import (
+	"fmt"
+	"sort"
+
+	"github.com/CircleCI-Public/circleci-cli/git"
+)
 
 // CircleCI provides various `<< pipeline.x >>` values to be used in your config, but sometimes we need to fabricate those values when validating config.
 type Values map[string]string
 
-
 func FabricatedValues() Values {
+
+	revision := git.Revision()
```

Comment:
It initially threw me off that we use this value for both `git.revision` and `git.base_revision` but I think it may make sense. The reality is the working directory may have state that isn't part of any commit, and in that case the current `HEAD` really is the "previous" revision. Plus I think we're only aiming for "realism" here, not 100% accuracy.